bundle/ezpublish_legacy/ngadminui/design/ngadminui/templates/content/view/versionview.tpl

```diff
@@ -28,6 +28,79 @@
+                    <option value="{$version.language_list[0].language_code}" selected="selected" disabled="disabled">
+                        {if $version.language_list[0].locale.is_valid}
+                            <img src="{$version.language_list[0].language_code|flag_icon}" width="18" height="12" alt="{$version.language_list[0].language_code}" style="vertical-align: middle;" /> {$version.language_list[0].locale.intl_language_name|shorten( 16 )}
+                        {else}
+                            {'%1 (No locale information available)'|i18n( 'design/admin/content/view/versionview',, array( $version.language_list[0].language_code ) )}
+                        {/if}
+                     </option>
+                {/if}
+                </select>
+            </div>
+        {/if}
+
+        {* Location *}
+        {section show=$version.node_assignments|count|gt( 0 )}
+        <div class="form-group pull-left">
+            <label class="hidden">{'Location'|i18n( 'design/admin/content/view/versionview' )}:</label>
+            <select class="form-control inline" name="SelectedPlacement">
+            {section show=$version.node_assignments|count|gt( 1 )}
+            {section var=Locations loop=$version.node_assignments}
+                <option value="{$Locations.item.id}" {if eq( $Locations.item.id, $placement )}selected="selected"{/if}>&nbsp;{$Locations.item.parent_node_obj.name|wash}</option>
+            {/section}
+            {section-else}
+                <option  value="{$version.node_assignments[0].id}" selected="selected" disabled="disabled">&nbsp;{$version.node_assignments[0].parent_node_obj.name|wash}</option>
+            {/section}
+            </select>
+        </div>
+        {/section}
+        <div class="form-group pull-left">
+            <label class="hidden">{'SiteAccess'|i18n( 'design/admin/content/view/versionview' )}:</label>
+            <select class="form-control inline" name="SelectedSiteAccess">
+            {if $site_access_locale_map|count|gt( 1 )}
+                {foreach $site_access_locale_map as $related_site_access => $related_site_access_locale}
+                    <option value="{$related_site_access}" {if eq( $related_site_access, $siteaccess )}selected="selected"{/if}>&nbsp;{$related_site_access|wash}</option>
+                {/foreach}
+            {else}
+                <option value="{$site_designs[0]}" selected="selected" disabled="disabled">&nbsp;{$site_designs[0]|wash}</option>
+            {/if}
+            </select>
+            <input class="button" type="submit" name="ChangeSettingsButton" value="{'Update view'|i18n( 'design/admin/content/view/versionview' )}" title="{'View the version that is currently being displayed using the selected language, location and design.'|i18n( 'design/admin/content/view/versionview' )}" />
+        </div>
+        <div class="form-group pull-left">
+            {if or( and( eq( $version.status, 0 ), $is_creator, $object.can_edit ),
```

Comment:
I would wrap the entire form with this condition, making it show up only when coming to the preview page from edit interface, because now when you open a preview page from the full view, you get the duplicated forms for selecting language and siteaccess (except parent location selection, but I don't know why it was removed originally):

![image](https://user-images.githubusercontent.com/362286/42496588-db66397c-8426-11e8-98c1-e5bd03f6f51b.png)

Reply:
Infact, it would be best I think that the new form you added reuses the design and location from the old form, and only retain "Back to edit" and "Publish" buttons as a separate form that shows up only in edit mode.

What do you think @goranNetgen ?

In any case, I think we will need to apply some design changes to this once we merge this functional part, to match the rest of Netgen Admin UI design.

Reply:
Yes, it would be best if the look and feel stayed the same but maybe we can take care of that once we merge it.